src/android/AudioHandler.java

```diff
@@ -55,6 +59,16 @@ Licensed to the Apache Software Foundation (ASF) under one
     private int origVolumeStream = -1;
     private CallbackContext messageChannel;
 
+
+    public static String [] permissions = { Manifest.permission.RECORD_AUDIO, Manifest.permission.WRITE_EXTERNAL_STORAGE};
+    public static int RECORD_AUDIO = 0;
+    public static int WRITE_EXTERNAL_STORAGE = 1;
+
+    public static final int PERMISSION_DENIED_ERROR = 20;
```

Comment:
Does the magic number need to be documented? I'm not very familiar with error handling and how Cordova deals with error codes.